src/pydata_sphinx_theme/__init__.py

```diff
@@ -1084,6 +1085,43 @@ def setup_translators(app):
             app.set_translator(name, translator, override=True)
 
 
+# ------------------------------------------------------------------------------
+# customize method for logo management
+# ------------------------------------------------------------------------------
+
+
+def setup_logo_path(
+    app: Sphinx, pagename: str, templatename: str, context: dict, doctree: nodes.Node
+) -> None:
+    """Set up relative paths to logos independant from the version"""
+
+    # get informations from the context
+    pathto = context.get("pathto")
+    logo = context.get("logo")
+    image_light = context.get("theme_logo", {}).get("image_light")
+    image_dark = context.get("theme_logo", {}).get("image_dark")
+
+    # create theme_options["logo"] if not existing
+    if "theme_logo" not in context:
+        context["theme_logo"] = {}
+
+    # create the logo_url
+    if logo and not isurl(logo):
+        logo = pathto(f"_static/{logo}", resource=True)
```

Comment:
here (and below) you're hard-coding `_static` but isn't there a config variable `html_static_path`?  Should we be using that here?

Reply:
`html_static_path` defines the source folders for static files but the target one is always called `_static`. As here we write the HTML `src` attributes, we can safely hard code "_static".

Reply:
ah good points - this is the _output_ path, not the _input_ source file folders 👍

Reply:
👍🏻